source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB specific information to users,
+supplemented by links to MongoDB Documentation. The |copilot| is
+knowledgeable of the latest MongoDB Documentation and uses the
+MongoDB Docs Chatbot API to link users to information in addition to
+the textual response. 
+
+Example
+~~~~~~~
+
+Consider the following prompt that asks for information on creating an
+index in the GitHub Copilot chat:
+
+.. code-block:: javascript
+   :copyable: false
+
+   @MongoDB /docs how do I create an index?
+
+The chat first provides the following MongoDB documentation:
+
+- `Create an Index <https://www.mongodb.com/docs/manual/core/indexes/create-index/>`__
+- `createIndexes <https://mongodb.com/docs/manual/reference/command/createIndexes/>`__
+- `Getting Started with MongoDB and Java - CRUD Operations Tutorial <https://www.mongodb.com/developer/languages/java/java-setup-crud-operations>`__
+- `db.collection.createIndex() <https://mongodb.com/docs/manual/reference/method/db.collection.createIndex/>`__
+   
+The |copilot| also returns a textual response with information about
+creating an index:
+
+.. code-block:: javascript
+   :copyable: false
+
+   To create an index in MongoDB, you can use the createIndex() method
+   in the MongoDB Shell or the equivalent method for your specific
+   driver. Here are examples for different programming languages:
+
+   MongoDB Shell
+
+      db.collection.createIndex({ <key and index type specification> }, { <options> })
+
+   C#
+
+      collection.CreateIndex(IndexKeys<collection>.<key and index type
+      specification>, <options>);
+      
+   ...
+
+   Indexes support efficient execution of queries in MongoDB. If your
+   application is repeatedly running queries on the same fields,
+   creating an index on those fields can significantly improve
+   performance. However, keep in mind that while indexes improve query
+   performance, they can negatively impact write operations due to the
+   additional overhead of maintaining the index.
+
+.. figure:: /images/copilot-docs.png
+   :figwidth: 700px
+   :alt: Screenshot of copilot providing information on creating an index
```

Comment:
I'm not sure we need the screenshot here, if it's just showing what we've described above?

Reply:
I'll check in with Garaub about this one - he originally asked for screenshots on the /query page to illustrate the UI

Reply:
@GaurabAryal do you have preference here?

Reply:
Yeah i think since this is such a new product, it is better to have the visuals in the docs. Let's keep it for now.